radio/src/targets/flysky/aux_serial_driver.cpp

```diff
@@ -66,7 +66,9 @@ void auxSerialInit(unsigned int mode, unsigned int protocol)
         break;
       }
 #endif
+#if defined(TELEMETRY_FRSKY)
```

Comment:
This definition will be removed in the future (as it is in OpenTX) and should not be added anywhere. It will make backporting more difficult.